src/roles/common/tasks/setup-kvm.yml

```diff
@@ -38,6 +38,7 @@
 
   - name: If RedHat, install packages for RedHat OS family distros
     yum:
+      lock_timeout: 100
```

Comment:
Can you move this to after the list of packages? I think for consistency it's better to have the package list first and then additional parameters after that